index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/3.0.3/normalize.css" media="screen" title="no title" charset="utf-8">
+    <link rel="stylesheet" href="css/styles.css" media="screen" title="no title" charset="utf-8">
+    <link href='https://fonts.googleapis.com/css?family=Source+Sans+Pro:400,900,600,900italic|Source+Serif+Pro:400,600,700' rel='stylesheet' type='text/css'>
+    <title>JavaScript</title>
+  </head>
+  <body>
+    <header>
+      <nav>
+        <h1>javaScript</h1>
+        <ul>
+          <li><a href="index.html" class="active-link">About</a></li>
+          <li><a href="portfolio.html">Experience</a></li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <section class="main-left">
+        <h2>About</h2>
+        <p class="first-p">
+          JavaScript  is a high-level, dynamic, untyped, and interpreted programming language. It has been standardized in theECMAScript language specification. Alongside HTML and CSS, it is one of the three essential technologies of World Wide Web content production; the majority of websites employ it and it is supported by all modern Web browsers without plug-ins. JavaScript is prototype-based with first-class functions, making it a multi-paradigm language, supporting object-oriented,[8] imperative, and functional programming styles. It has an API for working with text, arrays, dates and regular expressions, but does not include any I/O, such as networking, storage, or graphics facilities, relying for these upon the host environment in which it is embedded.[7]
+        </p>
+        <p>
+          Despite some naming, syntactic, and standard library similarities, JavaScript and Java are otherwise unrelated and have very different semantics. The syntax of JavaScript is actually derived from C, while the semantics and design are influenced by the Self and Scheme programming languages.
+        </p>
+        <p>
+          JavaScript is also used in environments that are not Web-based, such as PDF documents, site-specific browsers, and desktop widgets. Newer and faster JavaScript virtual machines (VMs) and platforms built upon them have also increased the popularity of JavaScript for server-side Web applications. On the client side, JavaScript has been traditionally implemented as an interpreted language, but more recent browsers perform just-in-time compilation. It is also used in game development, the creation of desktop and mobile applications, and server-side network programming with runtime environments such as Node.js.
+        </p>
+        <p>
+          Source <a href="https://en.wikipedia.org/wiki/JavaScript">Wikipedia</a>
+        </p>
+      </section>
+      <section class="main-right">
+        <figure class="main-figure">
```

Comment:
@chrislindor I have nothing but good things to say. Great usage of the `<figure>` element here with the `<figcaption>`.